code.html

```diff
@@ -1,13 +1,13 @@
 <!DOCTYPE html>
 <head>
-<link rel="stylesheet" type="text/css" href="css/bootstrap.css"/>
+<link rel="stylesheet" type="text/css" href="css/bootstrap.css"/><!-- -->
 </head>
-<header style ="background-color:gold">
-	<body style="color:auto";margin="960,auto";>
-		<h1 style="margin: 960px,auto">
+<header style ="background-color:gold"><!-- -->
+	<body style="color:auto";margin="960,auto";><!-- -->
+		<h1 style="margin: 960px,auto"><!-- -->
 			<p style="margin:960">May the force be with you</p>
```

Comment:
please do not write inline styles. use embedded styles in head. also you must have a unit of measurement for margin. in this case, px